src/App.tsx

```diff
@@ -8,23 +8,45 @@ import { SnakeStates } from './Snake';
 function App(): JSX.Element {
   const [pokemon, setPokemon] = useState<Pokemon['id']>(1);
   const [isOnGame, setIsOnGame] = useState(false);
+  const [isPaused, setIsPaused] = useState(true);
 
   const nextClick = () => {
     setIsOnGame(true);
   };
 
+  const pauseHandler = () => {
+    setIsPaused((prev) => !prev);
+  };
+
+  if (isOnGame) {
+    return (
+      <div className='App'>
+        {isPaused ? (
+          <div className={'pause'}>
+            <button className={'play'} onClick={pauseHandler}>
+              {'\u25B6'}
+            </button>
+          </div>
+        ) : (
+          <div className={'play'}>
+            <button className={'pause'} onClick={pauseHandler}>
+              {'\u2759 \u2759'}
+            </button>
+          </div>
+        )}
+        <SnakeStates pokemonId={pokemon} isPaused={isPaused} setIsOnGame={setIsOnGame} />
```

Comment:
Components produce events. That is why they should not have setters as parameters. Events syntax describes much better what a component is going to do.

Better, but not very good variant:
```suggestion
        <SnakeStates pokemonId={pokemon} isPaused={isPaused} onChangeIsOnGame={setIsOnGame} />
```

Best variant:
```suggestion
        <SnakeStates pokemonId={pokemon} isPaused={isPaused} onEndGame={() => setIsOnGame(false)} />
```